Tasks/Common/packaging-common/nuget/NuGetToolRunner2.ts

```diff
@@ -46,6 +47,12 @@ function prepareNuGetExeEnvironment(
     let originalCredProviderPath: string = null;
     let envVarCredProviderPathV2: string = null;
 
+    // Set NUGET_PLUGINS_CACHE_PATH to work aroud the NuGet issue with long paths: https://github.com/NuGet/Home/issues/7770
```

Comment:
Please make this workaround configurable via variable, in case it needs to be disabled.